### DISCUSSION TITLE
Easier way to suggest edits?

### DISCUSSION THREAD

**Discussion by jdevinemt** [symfony/symfony#51469] [Symfony Book: The Fast Track]
I've been migrating our company's application to the Symfony framework over the past year or so, and am a huge fan of the project. The documentation is some of the best I've seen. I recently started going through the book, [Symfony: The Fast Track](https://symfony.com/book) as a way to familiarize myself with Platform.sh since our company is looking at possibly migrating to that platform. 

As I've been going through the book, I've noticed a decent amount of errors, typos and outdated links. Luckily, I'm familiar enough with the framework and concepts that resolving errors myself is not an issue, but I would love to have the ability to edit the content as I'm going through it for submission upon completion. I wasn't able to find a repo for the book's content to fork from. I assume this is likely because it's a *book* and not *documentation* per se. However, it would be nice for the book have a process similar to submitting a pull request for the documentation.

Is there a preferred method for the submission of these edits? I had started a markdown file with the edits, but it was too tedious since I had to describe each change rather than just making it. I supposed I would have to describe the reasoning for changes anyway, but a lot of the changes are self-explanatory.

If there is a good way to do this, I'd love to provide my feedback to help improve the experience for other readers.

That being said. Although I've encountered these issues while following along with the book, it is really a phenomenal resource!

**Comment by chapterjason** [comment]
IIRC the documentation moved to platform.sh and is not maintained in symfony-docs anymore. Also, the docs link on the cloud page links to that.

- https://symfony.com/cloud/
- https://docs.platform.sh/guides/symfony.html
- https://github.com/platformsh/platformsh-docs/tree/main/sites/platform/src/guides/symfony

**Reply by jdevinemt** [reply]
I was referring to making edits to the book, ["Symfony: The Fast Track"](https://symfony.com/book), which is hosted on the Symfony site. I've edited my original post to be more clear regarding that.
